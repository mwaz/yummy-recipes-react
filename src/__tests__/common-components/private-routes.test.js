import React from 'react';
import { shallow, mount, render } from 'enzyme';
import Private from '../../containers/private';
import toJson, { shallowToJson } from 'enzyme-to-json';

describe('Login component', () => {
    const wrapper = shallow(<Private />);

    it('renders properly without crashing', () => {
        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });
});