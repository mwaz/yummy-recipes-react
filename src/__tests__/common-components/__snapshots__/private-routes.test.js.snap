// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login component renders properly without crashing 1`] = `
<Route
  render={[Function]}
/>
`;
